bbb-webhooks cannot be installed

<!--PLEASE DO NOT FILE ISSUES FOR GENERAL SUPPORT QUESTIONS.
This issue tracker is only for bbb development related issues.
For support of BBB installation problems ask in the forum: 
https://groups.google.com/forum/#!forum/bigbluebutton-setup -->

**Describe the bug**

Cannot install BBB package `bbb-webhooks` due to `npm ERR! `unsafe-perm` is not a valid npm option`. Initial investigation points to `npm` version troubles. I find `unsafe-perm` in npm's docs  https://docs.npmjs.com/cli/v6/using-npm/config?v=true#unsafe-perm on versions up to 7, but not after. Right now BBB 2.5, 2.6 use npm 8, and we're transitioning to version 9+ for BBB 2.7. 



**Console output**
```
root@droplet-8443:/usr/local/bigbluebutton# apt install bbb-webhooks 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following NEW packages will be installed:
  bbb-webhooks
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 5386 kB of archives.
After this operation, 19.1 MB of additional disk space will be used.
Get:1 https://ubuntu.bigbluebutton.org/focal-27-dev bigbluebutton-focal/main amd64 bbb-webhooks amd64 1:2.7-3 [5386 kB]
Fetched 5386 kB in 0s (29.1 MB/s)     
Selecting previously unselected package bbb-webhooks.
(Reading database ... 234471 files and directories currently installed.)
Preparing to unpack .../bbb-webhooks_1%3a2.7-3_amd64.deb ...
Unpacking bbb-webhooks (1:2.7-3) ...
Setting up bbb-webhooks (1:2.7-3) ...
npm ERR! `unsafe-perm` is not a valid npm option

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2023-06-09T10_07_19_200Z-debug-0.log
dpkg: error processing package bbb-webhooks (--configure):
 installed bbb-webhooks package post-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 bbb-webhooks
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

**Additional context**
Also reported in https://groups.google.com/g/bigbluebutton-dev/c/11RlUkG0yh8/m/PA9s6cg2AgAJ

--- a/build/packages-template/bbb-webhooks/after-install.sh
+++ b/build/packages-template/bbb-webhooks/after-install.sh
@@ -18,12 +18,6 @@ case "$1" in
     yq w -i $TARGET server.port "3005"
     yq w -i $TARGET hooks.getRaw "false"
 
-    cd /usr/local/bigbluebutton/bbb-webhooks
-    mkdir -p node_modules
-
-    npm config set unsafe-perm true
-    npm rebuild || true
-
     mkdir -p /var/log/bbb-webhooks/
     touch /var/log/bbb-webhooks/bbb-webhooks.log
     chown -R bigbluebutton:bigbluebutton /usr/local/bigbluebutton/bbb-webhooks /var/log/bbb-webhooks/

--- a/build/packages-template/bbb-webhooks/build.sh
+++ b/build/packages-template/bbb-webhooks/build.sh
@@ -26,7 +26,7 @@ find -maxdepth 1 ! -path . ! -name staging $(printf "! -name %s " $(cat .build-f
 
 pushd .
 cd staging/usr/local/bigbluebutton/bbb-webhooks/
-npm install --production
+npm install --unsafe-perm --production
 popd
 
 cp webhooks.nginx staging/usr/share/bigbluebutton/nginx/webhooks.nginx

